docker/docker-compose.yml

```diff
@@ -21,22 +21,19 @@ services:
       - default.env
       - ../dev.env
     depends_on:
-      - pygeoapi
+      - wis2box-api
 
-  pygeoapi:
-    container_name: pygeoapi
-    build:
-      context: ./pygeoapi
+  wis2box-api:
+    container_name: wis2box-api
+    image: ghcr.io/wmo-im/wis2box-api:latest
     env_file:
       - default.env
       - ../dev.env
-    environment:
-      - SCRIPT_NAME=/pygeoapi
     depends_on:
       - elasticsearch
+      - wis2box
```

Comment:
Why does the API now depend on wis2box?

Reply:
From discussion with Ben: this dependency now exists given the pygeoapi config is managed per se in wis2box.